server/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -676,9 +682,6 @@ public void inboundMessage(TcpChannel channel, InboundMessage message) {
     }
 
     public void inboundDecodeException(TcpChannel channel, Tuple<Header, Exception> tuple) {
```

Comment:
Can we remove this method now?

Reply:
There will be a follow up related to exception handling. Specifically CircuitBreaking will move inside the pipeline. But circuit breaking generates an exception that has to be handling by generating a message to send back.